openstack/designate/templates/etc-configmap.yaml

```diff
@@ -27,15 +27,6 @@ data:
 {{ include (print .Template.BasePath "/etc/_test.html.tpl") . | indent 4 }}
 {{- end }}
 
-{{- if .Values.tempest_enabled }}
```

Comment:
Shouldn't `designate/templates/seeds.yaml` also be cleaned from the tempest configuration? I see, that we have `tempest/designate-tempest/seed.yaml` with the same content.